app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= form_with model: [project, story, comment] do |form| %>
+  <%= form.text_area :body, rows: 6 %>
+  <%= form.submit class: "button green"%>
+  <%= link_to "Back", project_story_path(comment.story.project_id, comment.story_id), id:"back", class: "button" if action_name == "edit" %>
```

Comment:
```suggestion
  <%= link_to "Back", project_story_path(project, story), id: "back", class: "button" if action_name == "edit" %>
```

I think it looks cleaner to use the variables we already have for the path

also, if the update fails, the action_name is "update" and not "edit", so this button disappears

![image](https://github.com/fastruby/points/assets/188464/d9b31b20-5e06-40c5-ab2d-849a9bd86288)